examples/asset/custom_asset.rs

```diff
@@ -67,19 +107,55 @@ fn main() {
 #[derive(Resource, Default)]
 struct State {
     handle: Handle<CustomAsset>,
+    other_handle: Handle<CustomAsset>,
+    blob: Handle<Blob>,
     printed: bool,
 }
 
 fn setup(mut state: ResMut<State>, asset_server: Res<AssetServer>) {
+    // Recommended way to load an asset
     state.handle = asset_server.load("data/asset.custom");
+
+    // File extensions are optional, but are recommended
```

Comment:
Why are they recommended?

Reply:
Just a general tip regarding file management. Most file managers prefer file extensions as a way to determine default handlers. Specifically to Bevy, since file extensions are used as the last-resort for `Asset` disambiguation, it's good to try and have unique file extensions for asset types if possible. I could amend this comment to provide some explanation?

Reply:
> since file extensions are used as the last-resort for Asset disambiguation

Yes I was hoping for a qualification like this such that users can know why something is recommended